specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,3078 @@
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "projectKind",
+            "description": "The project type.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "conversation",
+              "orchestration"
+            ]
+          },
+          {
+            "in": "query",
+            "name": "top",
+            "description": "The number of configs to be returned.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "The offset in the response.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxpagesize",
+            "description": "The maximum number of configs per page.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of all trained models.",
+            "schema": {
+              "$ref": "#/definitions/TrainingConfigVersionsList"
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "404": {
+            "description": "The project does not exist."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ConfusionMatrixObject": {
+      "required": [
+        "normalizedValue",
+        "rawValue"
+      ],
+      "type": "object",
+      "properties": {
+        "normalizedValue": {
+          "format": "float",
+          "description": "Represents normalized value in percentages.",
+          "type": "number"
+        },
+        "rawValue": {
+          "format": "float",
+          "description": "Represents raw value.",
+          "type": "number"
+        }
+      }
+    },
+    "CreateOrReplaceDeploymentJobObject": {
+      "required": [
+        "trainedModelLabel"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModelLabel": {
+          "description": "Represents the trained model label.",
+          "type": "string"
+        }
+      }
+    },
+    "DeploymentJobStatusObject": {
```

Comment:
Deployment job should return a snapshot of a deployment so that users can reason what has been deployed and what model is used to do the inference. Essentially, modelId should be returned.